src/proto/FunctionRpc.proto

```diff
@@ -282,12 +282,15 @@ message FunctionsMetadataRequest {
 }
 
 // Worker sends function metadata back to host
-message FunctionMetadataResponses {
+message FunctionMetadataResponse {
   // list of function indexing responses
-  repeated FunctionLoadRequest function_load_requests_results = 1;
+  repeated RpcFunctionMetadata function_load_requests_results = 1;
```

Comment:
Since the type is changed to `RpcFunctionMetadata` can we also change `function_load_requests_results` to something that matches and is less confusing?

Reply:
Agreed. Let's change the name here

Reply:
Something like `function_metadata_results` (as that seems to be the convention) would be more appropriate.

Reply:
Agreed. Will update here.